### ISSUE TITLE
[BUG] Autocomplete="off" on honeypot field currently not working in Chrome

### ISSUE THREAD

**Issue by alexej-d** [dachcom-digital/pimcore-formbuilder#333]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

Adding role="presentation" to the honeypot field fixes Chrome's behavior. Do you see any downsides in adding this attribute per default? Otherwise it is often not possible to send the form, when using autofill.

**Comment by solverat** [comment]
Mhm, what do you think about a configuration flag?

```yaml

form_builder:
    spam_protection:
        honeypot:
            # NEW:
            add_role_attribute: true        # this is the default value and will add the role="presentation" attribute
            # field_name: 'inputUserName' 
            # enable_inline_style: true 
```

I don't think that spambots can rely on this value, since the presentation role is a legit attribute for every other visible element, right?

**Comment by solverat** [comment]
implemented in #339

**Comment by alexej-d** [comment]
Thank you @solverat !
